src/Symfony/Component/OptionsResolver/OptionsResolver.php

```diff
@@ -1124,18 +1137,17 @@ private function verifyTypes(string $type, $value, array &$invalidTypes, int $le
     {
         if (\is_array($value) && '[]' === substr($type, -2)) {
             $type = substr($type, 0, -2);
-            $valid = true;
 
             foreach ($value as $val) {
                 if (!$this->verifyTypes($type, $val, $invalidTypes, $level + 1)) {
-                    $valid = false;
+                    return false;
```

Comment:
this should be reverted: continuing the loop allows populating the `$invalidTypes` variable.

Reply:
I understood, Wasn't sure about it so this was committed separately and will remove it. Thanks!